packages/workspace/src/generators/move/lib/move-project-configuration.spec.ts

```diff
@@ -190,6 +196,8 @@ describe('moveProjectConfiguration', () => {
   });
 
   it('should support moving a standalone project', () => {
+    setupWorkspace(2);
```

Comment:
This was because the function now throws when `standalone` is `true` but the workspace version is 1?

Reply:
Exactly. The test tree is created by default using v1, but right after the creation in the `beforeEach` we call `addProjectConfiguration` which was turning it into v2, but with the change on this PR, it will continue to be v1, so this test case fails if we don't specify the workspace to use v2.